Buildings/Controls/OBC/Utilities/PIDWithAutotuning/FirstOrderAMIGO.mo

```diff
@@ -330,13 +338,31 @@ at which point this block turns back to a PID controller but with tuned PID para
 <b>Note:</b> If an autotuning is ongoing, i.e., <code>inTunPro.y = true</code>,
 a request for performing autotuning will be ignored.
 </p>
+<h4>Guidance for settings the parameters</h4>
 <p>
-In addition, this block requires an asymmetric relay output, meaning <code>yHig - yRef &ne; yRef - yLow</code>.
-</p>
-<p>
-Besides, one need to adjust <code>deaBan</code> based on the response from the control loop.
-e.g., decrease <code>deaBan</code> when the system response is slow to facilitate the tuning process.
+The performance of the autotuning is affected by the parameters, including <code>r</code>, 
+<code>yRef</code>, <code>yLow</code>, and <code>deaBan</code>.
+<br>
+The following procedure can be used for determining the values of those parameters. 
 </p>
+<ol>
+<li>
+The <code>r</code> should be adjusted so that the input difference of the relay controller, <code>rel.yDif</code>, is within the range from 0 to 1.
```

Comment:
The range of typical control error <code>r</code> should be adjusted so that the difference output of the relay controller <code>rel.yDif</code> is within the range from 0 to 1.